app/javascript/components/lease_ups/LeaseUpApplicationsPage.js

```diff
@@ -367,7 +376,7 @@ const LeaseUpApplicationsPage = () => {
     onSelectAllApplications: handleSelectAllApplications,
     pages: state.pages,
     preferences: listingPreferences,
-    rowsPerPage: ROWS_PER_PAGE,
+    rowsPerPage: rowsPerPage(),
```

Comment:
question: How is this change from static to dynamic row count related to the loading bug?

Reply:
Part of the ticket is to move up the no data message (see comments) - the quickest way to do that is to shorten the empty view (the message shows up in the middle for react table)

Reply:
Makes sense, I guess you tried updating the the `.ReactTable .rt-noData {...` entry in the stylesheet but found this solution to be easier?

Reply:
... I did not, lemme try

Reply:
That is much better! Thanks @jimlin-sfgov !

Reply:
Nice, this also fixes the issue with other kinds of tables in the Partners app